library/coretests/tests/num/mod.rs

```diff
@@ -924,6 +924,117 @@ macro_rules! test_float {
                 assert!($inf.div_euclid($nan).is_nan());
                 assert!($nan.div_euclid($inf).is_nan());
             }
+            #[test]
+            #[cfg(not(bootstrap))]
+            fn floor() {
+                $fassert!((0.0 as $fty).floor(), 0.0);
+                $fassert!((0.0 as $fty).floor().is_sign_positive());
+                $fassert!((-0.0 as $fty).floor(), -0.0);
+                $fassert!((-0.0 as $fty).floor().is_sign_negative());
+                $fassert!((0.5 as $fty).floor(), 0.0);
+                $fassert!((-0.5 as $fty).floor(), -1.0);
+                $fassert!((1.5 as $fty).floor(), 1.0);
+                $fassert!(($max as $fty).floor(), $max);
+                $fassert!(($min as $fty).floor(), $min);
+                $fassert!(($min_pos as $fty).floor(), 0.0);
+                $fassert!((-$min_pos as $fty).floor(), -1.0);
+                $fassert!(($nan as $fty).floor().is_nan());
+                $fassert!(($inf as $fty).floor(), $inf);
+                $fassert!(($neginf as $fty).floor(), $neginf);
```

Comment:
The way we refer to these consts changed a bit with https://github.com/rust-lang/rust/pull/141571 so this will need a rebase and similar adjustments.